modules/apps/account/account-test/src/testFunctional/macros/Account.macro

```diff
@@ -1063,7 +1063,7 @@ definition {
 
 	@summary = "Default summary"
 	macro grantUserAccessToAccountAdmin(userEmailAddress = null) {
-		JSONRole.addRegularRole(roleTitle = "Access Account Permission");
```

Comment:
can we pass both the parameters (roleTitle AND roleKey with the same value?) - because otherwise I think the title remains empty and I don't know if in UI the tests will be able to catch the correct one

This for all the files changed here

Reply:
Hey @smottal thanks for the review! As I made [this modification](https://github.com/liferay-commerce/liferay-portal/pull/5284/commits/ab5e0b481761ab3ec96c289de90873d295b5bef2#diff-d0ca12e38865f81d50693f4f047df0ebc84845e66bf7e5127f9d993eda3a9cd8R40-R41) to the `JSONRole.addRegularRole` macro, I believe that we are just maintaining the previous behavior:

Call `JSONRoleAPI._addRole` using filled name and empty title. Am I missing something?